audeer/core/io.py

```diff
@@ -231,6 +232,30 @@ def create_archive(
         )
 
 
+def current_dir() -> str:
+    r"""Folder in which caller of this function is located.
+
+    When called from a file,
+    it returns the directory,
+    in which the file is stored.
+    When called in an interactive session,
+    it returns the current directory
```

Comment:
Usually, this is called "current working directory", e.g. https://docs.python.org/3/library/os.html#os.getcwd.
Should I change this here as well?

Reply:
Yes, I would name it "current working directory".

Reply:
I changed that already, but forgot to update the screen shot accordingly. I have fixed that now.